src/inc/switches.h

```diff
@@ -262,7 +262,7 @@
 #define FEATURE_STACK_SAMPLING
 #endif // defined (ALLOW_SXS_JIT)
 
-#if defined(_TARGET_ARM64_)
+#if defined(_TARGET_ARM64_) && !defined(FEATURE_CORECLR)
 #define FEATURE_INTERPRETER
```

Comment:
If you need to turn this off, it should be done for Unix only.

We are using the interpreter for bring up. Console "Hello world" is running with the interpreter enabled on Windows - when the JIT hits one of the not-yet-implemented paths, the interpreter is used as fallback.